### ISSUE TITLE
[Linux] New installer: can't access local I2C hardware anymore

### ISSUE THREAD

**Issue by UncleSamSwiss** [ioBroker/ioBroker#121]
For testing purposes I installed ioBroker on an empty Raspberry Pi (just the latest Raspbian minimal + SSH enabled) using your new installer script.

When I try to use the I2C bus with my [I2C adapter](https://github.com/UncleSamSwiss/iobroker.i2c), I can't find any I2C devices. Unfortunately the i2c-bus module I'm using doesn't give any errors.

But, if I simply start NodeJS on the Raspberry Pi and try to access I2C it works without issues. This works with the `pi` user, but also with `sudo -u iobroker node`.

I tried adding the `iobroker` user to the `i2c` group; it didn't help (also after a restart).

This is a **major issue** for anybody using my adapter - it will simply not work anymore for them.

This seems to go in the same direction as #116 

**How to test with just NodeJS:**
You need to have I2C enabled on the Pi, at least one I2C device connected to the Raspberry Pi and the device must be powered.
Console on Raspberry Pi:
`npm i i2c-bus`
`node`
Then in the NodeJS prompt:
`var i2c = require('i2c-bus');`
`var i2c1 = i2c.openSync(1);`
`i2c1.scanSync();`
Returns: `[118]`

**Current workaround:** (only for the very desperate)
1. stop the adapter instance in ioBroker.admin
2. open the shell and cd to `/opt/iobroker/node_modules/iobroker.i2c`
3. call `node i2c.js 0 debug --force`

**Comment by AlCalzone** [comment]
> but also with sudo -u iobroker node.

That I don't understand. There should be no difference between iobroker creating the process and you starting it manually as the iobroker user.

Do you know which permissions are necessary to access the i2c devices?

**Comment by AlCalzone** [comment]
And which command did you use to add iobroker to the i2c group?

**Comment by UncleSamSwiss** [comment]
OK, this is embarrassing...

After all the changes I did (as explained above) I did now a complete power cycle of the pi and after that it worked. I don't know which step it was because I'm certain I rebooted (`sudo reboot`) after adding the user to the group.

So, I think the only thing that is left to be done in this issue is actually add `iobroker` to the group `i2c` in the installer.

Sorry for the confusion!
